src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier_transports.php

```diff
@@ -317,5 +318,9 @@
             ->parent('notifier.transport_factory.abstract')
             ->tag('texter.transport_factory')
 
+        ->set('notifier.transport_factory.simple-textin', SimpleTextinTransportFactory::class)
+            ->parent('notifier.transport_factory.abstract')
+            ->tag('texter.transport_factory')
```

Comment:
Wait, this class is configured as `texter`, so it is supposed to support `SmsMessage`. `SmsMessage` has no possibility to add options...

Reply:
%^#$%.

What is the best approach?

1. Remove the options from all my texter bridges, or
2. Add option the SmsMessage?

Reply:
> %^#$%.

Sorry, I didn't spotted this earlier, but if there was a test case using options it would be even easier for you to spot this issue, too

> What is the best approach?

Lets wait for feedback in your PR
* #48503